ui/appui-react/src/appui-react/childwindow/InternalChildWindowManager.tsx

```diff
@@ -92,8 +94,12 @@ export class InternalChildWindowManager implements FrameworkChildWindows {
     element: React.FunctionComponentElement<any>,
     container: Element | DocumentFragment
   ) {
+    // If createRoot is passed in for React 18 we have to render differently
+    // than without it. React 17 vs React 18. We need to save the root to
+    // unmount it.
     if (this._createRoot) {
-      this._createRoot(container).render(element);
+      this._root = this._createRoot(container);
```

Comment:
We can have multiple windows open at the same time, this `root` should be stored in the `openChildWindows` object.

Reply:
resolved